ClientLib/src/main/java/com/o3dr/services/android/lib/drone/companion/solo/tlv/SoloPanoOptions.java

```diff
@@ -0,0 +1,157 @@
+package com.o3dr.services.android.lib.drone.companion.solo.tlv;
+
+import android.os.Parcel;
+import android.support.annotation.IntDef;
+
+import java.nio.ByteBuffer;
+
+/**
+ * Created by phu on 6/24/16.
+ */
+public class SoloPanoOptions extends SoloShotOptions {
```

Comment:
@phmagic doesn't look like you're using any of the methods/fields from the `SoloShotOptions` class. Why extending from it?